Docs/ProjectSpec.md

```diff
@@ -597,6 +597,7 @@ This is a convenience used to automatically generate schemes for a target based
 - [ ] **testTargets**: **[[Test Target](#test-target)]** - a list of test targets that should be included in the scheme. These will be added to the build targets and the test entries. Each entry can either be a simple string, or a [Test Target](#test-target)
 - [ ] **gatherCoverageData**: **Bool** - a boolean that indicates if this scheme should gather coverage data. This defaults to false
 - [ ] **disableMainThreadChecker**: **Bool** - a boolean that indicates if this scheme should disable disable the Main Thread Checker. This defaults to false
+- [ ] **stopOnEveryMainThreadCheckerIssue**: **Bool** - a boolean that indicates if this scheme should stop at every Main Thread Checker issue. This defaults to false
```

Comment:
Could you add docs to down below in `Schemes` as well https://github.com/yonaskolb/XcodeGen/blob/master/Docs/ProjectSpec.md#scheme

Reply:
Of course! Done